src/main/java/org/reaktivity/nukleus/tcp/internal/TcpConfiguration.java

```diff
@@ -20,12 +20,15 @@
 public class TcpConfiguration extends Configuration
 {
     public static final String MAXIMUM_BACKLOG_PROPERTY_NAME = "nukleus.tcp.maximum.backlog";
+    public static final String WINDOW_THRESHOLD_PROPERTY_NAME = "nukleus.tcp.window.threshold";
 
     /**
      * @see java.nio.channels.ServerSocketChannel#bind(java.net.SocketAddress, int)
      */
     private static final int MAXIMUM_BACKLOG_DEFAULT = 0;
 
+    private static final int WINDOW_THRESHOLD_DEFAULT = 50;
```

Comment:
Let's default to `0` to avoid any change in behavior, then we can assess the impact and potentially change to a different default later.